test/format/config.c

```diff
@@ -283,18 +294,24 @@ config_cache(void)
 	for (;;) {
 		max_dirty_bytes = ((g.c_cache * WT_MEGABYTE) / 10) * 4;
 		if (SIZE_ADJUSTMENT * g.c_threads *
-		    (g.c_intl_page_max + g.c_leaf_page_max) <= max_dirty_bytes)
+		    (g.intl_page_max + g.leaf_page_max) <= max_dirty_bytes)
 			break;
-		g.c_cache += g.c_threads * g.c_leaf_page_max;
+		++g.c_cache;
```

Comment:
Does this change make this loop really slow incrementing by one byte?

Reply:
`g.c_cache` is in units of MB, so it's not that slow.